google/cloud/spanner/client.h

```diff
@@ -603,6 +603,34 @@ class Client {
   StatusOr<CommitResult> Commit(Transaction transaction, Mutations mutations,
                                 Options opts = {});
 
+  /**
+   * Commits a write transaction with at-least-once semantics.
+   *
+   * Apply the given mutations atomically, using a single RPC, and therefore
+   * without replay protection.  That is, it is possible that the mutations
+   * will be applied more than once. If the mutations are not idempotent, this
+   * may lead to a failure (for example, an insert may fail with "already
+   * exists" even though the row did not exist before the call was made).
+   * Accordingly, this call may only be appropriate for idempotent, latency-
+   * sensitive and/or high-throughput blind writing.
+   *
+   * @note Prefer the `Commit` overloads if you want exactly-once semantics
+   *     or want to reapply mutations after a `kAborted` error.
+   *
```

Comment:
It would be nice to have a really simple example using this function.

Reply:
Good idea.  I'll add one in a follow-up.